src/Sylius/Bundle/AdminBundle/index.js

```diff
@@ -11,20 +11,35 @@ const path = require('path');
 const Encore = require('@symfony/webpack-encore');
 
 class SyliusAdmin {
+    /**
+     * Provide a light Webpack configuration for Sylius Admin
+     * All the stimulus stuff should be handled by the app.admin entrypoint
+     */
+    static getBaseWebpackConfig(rootDir) {
+        this._prepareWebpackConfig(rootDir);
+        Encore
```

Comment:
```suggestion
        this._prepareWebpackConfig(rootDir);
        
        Encore
```

Would be cool to have same syntax as at `getWebpackConfig`